### PR TITLE
Fix: Change unknown in idioms

### PR DESCRIPTION
**Pull request by D-Rosa99**
#### Description:

I changed the unknown label in idioms and put it the label depends on the language of the app

#### Linked issues:

Fixes [#92](https://trello.com/c/VeI3amf5/92-change-unknown-en-el-idioma)
#### Screenshots:
image

#### How to test:

- `git pull`
- `git checkout Fix/92-idioms`
- Head over to settings screen and change the language
- it should change the label and the language of settings

### CODE REVIEW

File: app/NavEntry.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ import ReportScreen from './views/DR/ReportScreen';
 const Tab = createBottomTabNavigator();
 
 function MainNavigation() {
+  const { t } = useTranslation();
```
Comment (inteljudajocu) [inline app/NavEntry.js:16]: Why t, the name is not descriptive

**Review by inteljudajocu — COMMENTED** [review]
Comments

File: app/NavEntry.js (hunk 1 above)
Comment (D-Rosa99) [reply app/NavEntry.js:16]: that is the standard in the project and i18n, t means translate, that's proper of the package

**Review by JoseFcoEstrella — APPROVED** [review]
LGTM,
Just solve the comment

File: app/NavEntry.js (hunk 1 above)
Comment (inteljudajocu) [reply app/NavEntry.js:16]: Can it be changed for us, so it'll be easier to read in the future

Comment (D-Rosa99) [reply app/NavEntry.js:16]: not a good idea, cause it's been use in many places in the project and not just in this ticket, also many projects and i18n docs is using it like that

**Review by RutsSantos — APPROVED** [review]
LGTM, but please merge develop into this branch

**Review by inteljudajocu — APPROVED** [review]

**Pull request merged**